### PR TITLE
Add Mogilefs collector

### PR DESCRIPTION
**Pull request by hrchu**
This collector collect stats from mogilefs tracker:

```
# telnet localhost 7001
Trying ::1...
Trying 127.0.0.1...
Connected to localhost.
Escape character is '^]'.
!stats
uptime 181491
pending_queries 0
processing_queries 1
bored_queryworkers 49
queries 4353158
times_out_of_qworkers 2
work_queue_for_delete 2
work_queue_for_replicate 0
work_sent_to_delete 336154
work_sent_to_replicate 274882
```

### CODE REVIEW

**Comment by shortdudey123** [comment]
Can you expand the tests with stub data? Also add documentation to the collector file and generate the doc file for the collector

**Comment by hrchu** [comment]
@shortdudey123 done, 
but I do not understand what's wrong with CI...
It's something wrong in the environment?
```
Collecting pyutmp (from -r .travis.requirements.txt (line 16))
  Using cached pyutmp-0.2.1.zip
    sh: 1: cython: not found
    Failed to build "pyutmp_linux.c" from "pyutmp_linux.pyx"
    "pyutmp_linux.pyx" is newer than "pyutmp_linux.c". Rebuilding "pyutmp_linux.c".
    + cython pyutmp_linux.pyx
    Complete output from command python setup.py egg_info:
    sh: 1: cython: not found
    
    Failed to build "pyutmp_linux.c" from "pyutmp_linux.pyx"
    
    "pyutmp_linux.pyx" is newer than "pyutmp_linux.c". Rebuilding "pyutmp_linux.c".
    
    + cython pyutmp_linux.pyx
    
    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-TgksUU/pyutmp
```

**Comment by shortdudey123** [comment]
Looks much better :)
Bombing out on `cython` is known issue so don't worry about it

Can you squash your commits down into 1?

**Comment by hrchu** [comment]
squash done

File: src/collectors/mogilefs/mogilefs.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * time
+
+
+"""
+import diamond.collector
+import telnetlib
+import time
+
+
+class MogstatsCollector(diamond.collector.Collector):
```
Comment (shortdudey123) [inline src/collectors/mogilefs/mogilefs.py]: class name should match the directory (i.e.`MogilefsCollector`)

File: src/collectors/mogilefs/mogilefs.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        })
+        return config_help
+
+    def get_default_config(self):
+        """
+        Returns the default collector settings
+        """
+        config = super(MogstatsCollector, self).get_default_config()
+        config.update({
+            'path':     'mogilefsd.stats'
```
Comment (shortdudey123) [inline src/collectors/mogilefs/mogilefs.py]: what is at the `mogilefsd` path? if nothing, this should probably go there and not at a sub path

**Comment by hrchu** [comment]
fix done

**Comment by shortdudey123** [comment]
LGTM

@jaingaurav can you review as well and merge?

**Pull request merged**